scripts/document.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+if [ -z `which jazzy` ]; then
+    echo "Installing jazzy…"
+    gem install jazzy
```

Comment:
What are the reasons `gem install` could fail?

Reply:
E.g. your gems require `sudo` 🙈

Reply:
Then it will give you error return code and script will fail. I will remove the second check.